src/patternfly/base/_common.scss

```diff
@@ -24,6 +24,14 @@
   @include pf-u-screen-reader; // for use with assistive technologies
 }
```

Comment:
```suggestion

  .pf-m-full-size {
    width: 100%;
    height: 100%;
  }
  
  &.pf-m-absolute {
    position: absolute;
  }
}
```

Reply:
@mattnolting nice agree, thought did you mean to suggest `&.pf-m-full-size`?